counsel.el

```diff
@@ -6548,7 +6579,10 @@ This is determined by `counsel-compile-local-builds', which see."
                                                      (mapconcat #'identity bldenv " ")
                                                      'font-lock-variable-name-face)))))
     (add-text-properties 0 (length cmd)
-                         `(srcdir ,srcdir blddir ,blddir bldenv ,bldenv) cmd)
+                         `(srcdir ,srcdir
+                                  buffer-file-name ,counsel-compile-buffer-file-name
+                                  blddir ,blddir bldenv ,bldenv)
```

Comment:
If you prefer, this can also be indented as follows:

```el
`( srcdir ,srcdir
   buffer-file-name ,counsel-compile-buffer-file-name
   blddir ,blddir bldenv ,bldenv )
```
This is a new convention in Emacs 28 introduced by the following commit: https://git.savannah.gnu.org/cgit/emacs.git/commit/?id=1b2a881c9b3ef9158cbf28a65b9e94c2dbc6cec2

```
*** A space between an open paren and a symbol changes the indentation rule.
The presence of a space between an open paren and a symbol now is
taken as a statement by the programmer that this should be indented
as a data list rather than as a piece of code.
```